pkg/service/schema.go

```diff
@@ -0,0 +1,96 @@
+package service
+
+// ParameterSchemas is a set of optional JSONSchemas that describe
+// the expected parameters for creation and update of instances and
+// creation of bindings.
+type ParameterSchemas struct {
+	ServiceInstances *InstancesSchema `json:"service_instance,omitempty"`
+	ServiceBindings  *BindingsSchema  `json:"service_binding,omitempty"`
+}
+
+// InstancesSchema represents a plan's schemas for creation and
+// update of an a service instance resource.
+type InstancesSchema struct {
```

Comment:
If we don't intend for anyone outside this package to instantiate this directly, we might consider not exporting this.